.container/README.md

```diff
@@ -85,5 +83,44 @@ stop and delete the container with the following command.
 docker compose down
 ```
 
-As mentioned in the previous section, all the local changes you made will be
-lost after deleting the container. So remember to save your progress!
+## Online Images
+For users who only want to have a quick tryout on CAMEL, we also provide the 
+pre-built images on
+[our GitHub Container Registry](https://github.com/camel-ai/camel/pkgs/container/camel).
+Considering the size of the image, we only offer the image with the basic 
+dependencies.
+
+Note that there are some key differences between the local development 
+image and the pre-built image that you should be aware of.
+1. The pre-built image is built upon the source code of each release of CAMEL. 
+   This means that they are not suitable for development, as they don't 
+   contain the git support. If you want to develop on CAMEL, please build 
+   the image by yourself according to the instructions above.
+2. The pre-built image only contains the basic dependencies for running the 
+   examples. If you want to run the examples that require additional 
+   dependencies, you need to install them according to the 
+   installation guide in CAMEL's [README](../README.md).
+3. The pre-built image doesn't contain the API keys. You need to set up the 
+   API keys by yourself in the container environment.
+4. The pre-built image does not support volume mounting. This means that all 
+   of your changes in the container will be lost when you delete the container.
+
+To quickly start a container with the pre-built image, you can use the
+following command.
+
+```bash
+docker run -it -d --name camel ghcr.io/camel-ai/camel:latest
```

Comment:
Just curious, if I can unify the overall operation by using docker compose, which may be more friendly to novices.

Reply:
I actually considered this but didn't do it at last, mainly out of the consideration that the online image is built directly from releases, and I think these images should be able to run without source code (actually this doc should also placed somewhere outside the source code but idk where for now). Therefore, I finally decided to discard the volume mounting and docker support for online images.

Reply:
Sure, I think when users have specific questions, we can consider it in more detail at that time.